PrimerSDK/Classes/User Interface/PCI/CardForm/CardFormViewModel.swift

```diff
@@ -31,6 +31,7 @@ class CardFormViewModel: CardFormViewModelProtocol {
         }
     }
     
+    // FIXME: Is this called only by the unit tests?
     func tokenize(instrument: PaymentInstrument, completion: @escaping (Error?) -> Void) {
```

Comment:
This entire view & view model should be deleted and replaced by FormView (this has not been done out of caution but should be fine)

Reply:
Removed